presto-orc/src/main/java/com/facebook/presto/orc/TupleDomainOrcPredicate.java

```diff
@@ -124,6 +126,9 @@ else if (type.getJavaType() == double.class && columnStatistics.getDoubleStatist
         else if (type.getJavaType() == Slice.class && columnStatistics.getStringStatistics() != null) {
             return createDomain(type, hasNullValue, columnStatistics.getStringStatistics());
         }
+        else if (FLOAT.equals(type) && columnStatistics.getDoubleStatistics() != null) {
```

Comment:
Currently, ORC uses DoubleStatistics for both Float and Double (see https://cwiki.apache.org/confluence/display/Hive/LanguageManual+ORC)